core-v1/src/main/scala/data/Fact.scala

```diff
@@ -105,11 +105,12 @@ object TypedFact {
     }
   }
 
-  def lens[A]: NamedLens.Id[TypedFact[A]] = NamedLens.id[TypedFact[A]]
-
-  def value[A]: NamedLens[TypedFact[A], A] = {
-    NamedLens[TypedFact[A], A](DataPath.empty.atField("value"), _.value)
-  }
+  // TODO: Switch to VariantLens
+//  def lens[A]: NamedLens.Id[TypedFact[A]] = NamedLens.id[TypedFact[A]]
+//
+//  def value[A]: NamedLens[TypedFact[A], A] = {
+//    NamedLens[TypedFact[A], A](DataPath.empty.atField("value"), _.value)
+//  }
```

Comment:
I don't think I really use these methods. I mostly rely on the expression builders in both algebras / DSLs.